### DISCUSSION TITLE
Use siteName.github.io instead of username.github.io/siteName/

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#43374] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way I can host a website, that it's name it's not my username, but at the same time the link to be short?

for example, I have a webpage named **mitkovtori.github.io**, I also have website named **mitkovtori.github.io/Mitko.Vtori-World/**.

Is there a way to name the second website **Mitko.Vtori-World.github.io** without creating a host account named Mitko.Vtori-world. or using custom domain.

**Comment by yoannchaudet** [comment]
Hello,

This is not possible. If you want a shorter name, other that what we provide on github.io, you can setup a [custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site).

**Comment by ttran278** [comment]
can i change my username? and if yes how I can change it? thank you

**Reply by MitkoVtori** [reply]
go to your profile, click on your profile image and then the first thing from top to bottom is your username, change it from there.
